src/main/kotlin/com/mdrobnak/lalrpop/psi/util/LpPsiUtil.kt

```diff
@@ -0,0 +1,10 @@
+package com.mdrobnak.lalrpop.psi.util
+
+import com.mdrobnak.lalrpop.psi.LpGrammarParam
+
+class LpPsiUtil {
+    companion object {
+        @JvmStatic
+        fun getName(grammarParam: LpGrammarParam): String = grammarParam.firstChild.text
```

Comment:
I don't think making a util function for this is necessary, especially as it's very specific to the type.

Reply:
I just don't feel comfortable using `firstChild` / `lastChild`/ indexing directly into `children`

Reply:
Your resolution of using an extension method looks good :+1: